ballsdex/core/bot.py

```diff
@@ -419,7 +420,7 @@ async def send(content: str):
             if isinstance(error, app_commands.CommandOnCooldown):
                 await send(
                     "This command is on cooldown. Please retry "
-                    f"in {math.ceil(error.retry_after)} seconds."
+                    f"in <t:{time.time() + math.ceil(error.retry_after)}:R> seconds."

```

Comment:
A few errors in the format, just had to group everything in the `math.ceil` and remove the words around it (Discord already writes "in x seconds").
<img width="676" alt="image" src="https://github.com/Ballsdex-Team/BallsDex-DiscordBot/assets/23153234/c4bc62e5-5bcb-4e52-bfec-52b2b0b28d80">


```suggestion
                    f"<t:{math.ceil(time.time() + error.retry_after)}:R>."
```

Reply:
Commited :D